frontend/app_v2/src/components/Alphabet/AlphabetPresentationSelected.js

```diff
@@ -32,10 +32,10 @@ const AlphabetPresentationSelected = ({ selectedData }) => {
             mb-5"
       >
         {title}
-        {src && (
+        {relatedAudio?.[0] && (
           <div className="ml-2">
             <AudioMinimal.Container
-              src={src}
+              src={`/nuxeo/${relatedAudio?.[0].url}`}
```

Comment:
```suggestion
              src={`/nuxeo/${relatedAudio?.[0].url}`}
```
We should avoid hardcoded paths where possible (or create URL helpers).